test/lambda/mongodb/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "mongodb",
-  "version": "1.0.0",
+  "version": "6.7.0",
```

Comment:
Why did this change?

We should probably change the name of this, set `private: true`, and remove the innaccurate license if we're cleaning up here.

Reply:
What is the reasoning behind keeping version at 1.0.0? I'm unfamiliar with lambdas, but I figured since we're updating dependencies for this folder and this is the only one, I'd update to be the most recent version of the driver.

Reply:
This is the version of "this" (as in this subdirectory) package it doesn't relate to the driver. This isn't really a package but managing dependencies without package.json is pretty tough and it requires you to define a version and name, even if it doesn't really have a version nor a name.